Vagrantfile

```diff
@@ -22,7 +23,7 @@ Vagrant.configure("2") do |config|
     vb.customize ["modifyvm", :id, "--cpus", 2]
     vb.customize ["modifyvm", :id, "--ioapic", "on"]
     vb.name = "geowave-vagrant"
-    vb.gui = true
+#    vb.gui = true
```

Comment:
I think most people just use 'vagrant ssh'.  For me it's just annoying to have the window popup...sorry, didn't actually mean to commit that one.